src/asmcnc/comms/router_machine.py

```diff
@@ -1053,18 +1053,144 @@ def disable_theateam(self):
     """
 
     def convert_from_110_to_230(self, rpm_green):
-        if float(rpm_green) != 0:
-            v_green = (float(rpm_green) - 9375)/1562.5
-            rpm_red = (2187.5*float(v_green)) + 3125
-            return float(rpm_red)
-        else: return 0
+        # DEPRECATED
+        #
+        # if float(rpm_green) != 0:
+        #     v_green = (float(rpm_green) - 9375)/1562.5
+        #     rpm_red = (2187.5*float(v_green)) + 3125
+        #     return float(rpm_red)
+        # else: return 0
+        return float(rpm_green)
 
     def convert_from_230_to_110(self, rpm_red):
-        if float(rpm_red) != 0:
-            v_red = (float(rpm_red) - 3125)/2187.5
-            rpm_green = (1562.5*float(v_red)) + 9375
-            return float(rpm_green)
-        else: return 0
+        # DEPRECATED
+        #
+        # if float(rpm_red) != 0:
+        #     v_red = (float(rpm_red) - 3125)/2187.5
+        #     rpm_green = (1562.5*float(v_red)) + 9375
+        #     return float(rpm_green)
+        # else: return 0
+        return float(rpm_red)
+
+    def correct_rpm_for_120(self, target_rpm):
+        """
```

Comment:
Perfect usage of doc strings - would say for consistency in the future use style seen in config_loader.py as I believe that’s the standard. But happy to merge.